packages/cli/helpers/copy-prisma-client.ts

```diff
@@ -1,23 +1,25 @@
-import fg from 'fast-glob'
 import { copySync } from 'fs-extra'
+import packlist from 'npm-packlist'
```

Comment:
Is `npm-packlist` as performant (for wach mode) as the previous solution? I remeber trying to use npm's arborist and it was pretty slow. So I think as long as this dep does not use that, it should be fine.

Reply:
I see no impact on build/ci times